### PR TITLE
Remove duplicate descriptions of the `--method` flag

### PR DESCRIPTION
**Pull request by naftali-g**
Discovered while working on CERT-6639

### CODE REVIEW

File: docs/prover/cli/options.md [hunk 1]
```diff
@@ -647,11 +647,7 @@ Options that help reduce the running time
 
 ### `--method`
 
-**What does it do?**
-Parametric rules will only verify the method with the given signature, instead of all public and external methods of the contract. Note that you will need to wrap the method's signature with quotes, as the shell doesn't interpret parenthesis correctly otherwise.
-
-**When to use it?**
-When you are trying to solve/understand a counterexample of a parametric rule on a specific method.
+See {ref}`--method`
```
Comment (yoav-el-certora) [inline docs/prover/cli/options.md:650]: The only issue imo is that both here and in the [--method](https://docs.certora.com/en/latest/docs/prover/cli/options.html#method) option, the example we provide is string and not a list.

Comment (naftali-g) [reply docs/prover/cli/options.md:656]: added an example of command-line multiple methods. For the conf case it;s just like all the other flags we have that support lists, I don;t think we need any special docs for it

File: docs/prover/cli/options.md [hunk 2]
```diff
@@ -647,11 +653,7 @@ Options that help reduce the running time
...
 
 **Example**
```
Comment (urikirsh) [inline docs/prover/cli/options.md:658]: Not sure if this is needed - there is an example in the link, and better to not keep several references which might get out of sync

**Review by urikirsh — APPROVED** [review]

File: docs/prover/cli/options.md (hunk 2 above)
Comment (naftali-g) [reply docs/prover/cli/options.md]: Yah I missed this. Removing it as well.

File: docs/prover/cli/options.md [hunk 3]
```diff
@@ -156,6 +156,12 @@ the contract or the spec to rerun, we can just rerun on the `deposit` method:
 certoraRun --method 'deposit(uint)'
 ```
 
+If we discover a counterexample in several methods, we could rerun just those:
+
+```sh
+certoraRun --method 'deposit(uint)' --method 'transfer(address,uint256)'
```
Comment (johspaeth) [inline docs/prover/cli/options.md:162]: @naftali-g shouldn't this be either
```suggestion
certoraRun --method ['deposit(uint)', 'transfer(address,uint256)']
```

**Review by johspaeth — CHANGES_REQUESTED** [review]
With the changes (that have landed in https://github.com/Certora/EVMVerifier/pull/6164) - I think we need to update some of the examples - see comment below?

File: docs/prover/cli/options.md (hunk 3 above)
Comment (naftali-g) [reply docs/prover/cli/options.md:162]: No, the way to specify multiple methods from command line is with multiple usages of the `--method` flag. In the conf file it's with square brackets, like any list.

Comment (johspaeth) [reply docs/prover/cli/options.md:162]: Thanks for clarification!

**Review by johspaeth — APPROVED** [review]
Approving after clarification.

**Pull request merged**